clusterman/autoscaler/pool_manager.py

```diff
@@ -437,8 +438,6 @@ def _get_prioritized_killable_nodes(self) -> List[ClusterNodeMetadata]:
     def _is_node_killable(self, node_metadata: ClusterNodeMetadata) -> bool:
         if node_metadata.agent.state == AgentState.UNKNOWN:
             return False
-        elif not node_metadata.agent.is_safe_to_kill:
```

Comment:
Why did this get deleted?

Reply:
This was a change made by @chlgit when implementing kube drain if you can chime in here.

Reply:
@enochtangg I do remember this and I don't see it from my [PR](https://github.com/Yelp/clusterman/pull/12/files). Could you double check this?

Reply:
Weird, the changes I pulled had all references of `safe_to_kill` removed. I'll add it back it in, thanks!